File: 2194-cells-in-a-range/range.py
class Solution:
    def cellsInRange(self, s: str) -> List[str]:
        rows = s.split(':')
        startLetter = ord(rows[0][0])
        endLetter = ord(rows[1][0])
    
        res = []
        for i in range(endLetter-startLetter+1):
            letter = chr(startLetter+i)
            for j in range(int(rows[1][1]) - int(rows[0][1])+1):
                res.append(letter+str(j+int(rows[0][1])))

        return res
        

File: 2194-cells-in-a-range/range.ts
function cellsInRange(s: string): string[] {
    let rows = s.split(':')
    let startLetter = rows[0].charCodeAt(0)
    let endLetter = rows[1].charCodeAt(0)
    
    let res = []
    for (let j = Number(startLetter); j <= Number(endLetter); j++){
        let letter = String.fromCharCode(j)
        for (let i = Number(rows[0][1]); i <= Number(rows[1][1]); i++ ){
                res.push(letter+i)
        }
    }

    return res
    
};